internal/common/common.go

```diff
@@ -285,6 +285,25 @@ func PathExists(filename string) bool {
 	return false
 }
 
+func PathExistsWithContents(filename string) bool {
+	if _, err := os.Stat(filename); err != nil {
+		return false
+	}
+
+	f, err := os.Open(filename)
+	if err != nil {
+		return false
+	}
+	defer f.Close()
+
+	r := bufio.NewReader(f)
+	_, err = r.ReadString('\n')
```

Comment:
please use r.ReadSlice('\n') here, this would not create an unused string.Builder